.github/workflows/ci_integration_tests.yaml

```
@@ -0,0 +1,59 @@
+name: Test with Home Assistant
+
+on:
+  pull_request:
+    branches:
+      - main
+      - dev
+  push:
+    branches:
+      - main
+      - dev
+
+jobs:
+  test_with_home_assistant:
+    name: Integration tests with Home Assistant
+    strategy:
+      matrix:
+        channel: [stable, beta, dev]
+        with_interation: ["yes", "no"]
```

```suggestion
        with_interation: [yes, no]
```
Maybe without quotes :shrug:  (it's currently not working)